README.md

```diff
@@ -216,13 +229,15 @@ _Note: Don't name the `static` directory anything else. The name is required and
 
 ### Populating `<head>`
 
-<p><details>
+<details>
   <summary><b>Examples</b></summary>
   <ul>
     <li><a href="./examples/head-elements">Head elements</a></li>
     <li><a href="./examples/layout-component">Layout component</a></li>
   </ul>
-</details></p>
+</details>
+
+<p></p>
```

Comment:
But it might look weird:

![image](https://user-images.githubusercontent.com/3676859/46246512-9bbb8700-c431-11e8-8c17-9da54e15cb95.png)

is that okay?